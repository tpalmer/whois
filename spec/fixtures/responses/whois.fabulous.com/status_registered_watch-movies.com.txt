

watch-movies.com is FOR SALE.
Please visit www.watch-movies.com for price and purchasing.

Domain watch-movies.com:
  Protopixel Pty Ltd
  P O Box 757
  Fortitude Valley, QLD 4006 AU

Administrative contact:
Technical contact:
Billing contact:
  Protopixel Pty Ltd
  Domain Hostmaster
  domain_hostmaster@protopixel.com
  P O Box 757
  Fortitude Valley, QLD 4006 AU
  Phone: +61.73007005
  Fax: +61.73007001

Record dates:
  Record created on: 2002-06-24 05:52:39 UTC
  Record modified on: 2011-06-25 07:10:30 UTC
  Record expires on: 2012-06-24 UTC

Nameservers:
  ns4.fabulous.com
  ns3.fabulous.com

Note: Automated collection of data from this database is strictly prohibited.

