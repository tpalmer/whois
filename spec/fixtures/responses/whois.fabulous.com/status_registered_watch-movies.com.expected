#status
  should: %s == :registered

#available?
  should: %s == false

#registered?
  should: %s == true


#created_on
  should: %s CLASS(time)
  should: %s == Time.parse("2002-06-24 05:52:39 UTC")

#updated_on
  should: %s CLASS(time)
  should: %s == Time.parse("2011-06-25 07:10:30 UTC")

#expires_on
  should: %s CLASS(time)
  should: %s == Time.parse("2012-06-24 UTC")


#registrar
  should: %s CLASS(registrar)
  should: %s.id           == nil
  should: %s.name         == "Fabulous.com"
  should: %s.organization == "Fabulous.com Pty Ltd"
  should: %s.url          == "http://www.fabulous.com"

#registrant_contacts
  should: %s CLASS(array)
  should: %s SIZE(1)
  should: %s[0] CLASS(contact)
  should: %s[0].type         == Whois::Record::Contact::TYPE_REGISTRANT
  should: %s[0].id           == nil
  should: %s[0].name         == nil
  should: %s[0].organization == "Protopixel Pty Ltd"
  should: %s[0].address      == "P O Box 757"
  should: %s[0].city         == "Fortitude Valley"
  should: %s[0].state        == "QLD"
  should: %s[0].zip          == "4006"
  should: %s[0].country_code == "AU"
  should: %s[0].phone        == nil
  should: %s[0].fax          == nil
  should: %s[0].email        == nil

#admin_contacts
  should: %s CLASS(array)
  should: %s SIZE(0)

#technical_contacts
  should: %s CLASS(array)
  should: %s SIZE(0)

#nameservers
  should: %s CLASS(array)
  should: %s SIZE(2)
  should: %s[0] CLASS(nameserver)
  should: %s[0].name == "ns4.fabulous.com"
  should: %s[0].ipv4 == nil
  should: %s[1] CLASS(nameserver)
  should: %s[1].name == "ns3.fabulous.com"
  should: %s[1].ipv4 == nil
